src/App.js

```diff
@@ -1,25 +1,69 @@
-import logo from './logo.svg';
+import React from 'react';
 import './App.css';
+import axios from "axios";
+import { BrowserRouter as Router, Routes, Route, NavLink } from 'react-router-dom';
+import HomePage from './components/HomePage';
+import ListBeers from './components/ListBeers';
+import NewBeer from './components/NewBeer';
+import RandomBeer from './components/RandomBeers';
+import SingleBeers from './components/SingleBeers';
+import { useEffect ,useState } from "react";
+
 
 function App() {
+
+  const [appArray, setAppArray] = useState();
+  const [randomArray, setRandomArray] = useState(null);
+
+
+  const baseURL= "https://ih-beers-api2.herokuapp.com/beers";
+  
+  const randomURL ="https://ih-beers-api2.herokuapp.com/beers/random";
+
+  useEffect(() => {
+    getBeerAPI();
+    getRandomBeerAPI();
+  },[]);
+  
+  const getBeerAPI = () => {
+    axios.get(`${baseURL}`)
+    .then(response => {
+      setAppArray(response.data)})
+      .catch(e => console.log(e));
+    };
+
+  const getRandomBeerAPI = () => {
+    axios.get(`${randomURL}`)
+    .then(response => {
+      setRandomArray([response.data])})
```

Comment:
I wouldn't use an Array to wrap that one, as you would get just one, so a little bit easier to get the information directly instead of passing forward as an array

```suggestion
      setRandomArray(response.data)})
```